### ISSUE TITLE
migration test cases failed in automtion environment for both rhels and sles against xcat 20180809.0615 build

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#5489]
xcat version : 20180809.0615 daily build

migration test cases failed in automtion environment for both rhels and sles.

The output of failed cases in rhels 7.5  
[rhels7.5-P8VMs-migration-testcase.log](https://github.com/xcat2/xcat-core/files/2276353/rhels7.5-P8VMs-migration-testcase.log)

The output of failed cases in sles12.2
[sles12.2-X86VMs-migration-testcase.log](https://github.com/xcat2/xcat-core/files/2276355/sles12.2-X86VMs-migration-testcase.log)

**Comment by robin2008** [comment]
@neo954   Would you like to take a look?

```
c910f03c09k11: Transaction Summary
c910f03c09k11: ================================================================================
c910f03c09k11: Upgrade  8 Packages
c910f03c09k11: 
c910f03c09k11: Total download size: 3.5 M
c910f03c09k11: Downloading packages:
c910f03c09k11: Public key for perl-xCAT-2.14.3-snap201808090615.noarch.rpm is not installed
c910f03c09k11: --------------------------------------------------------------------------------
c910f03c09k11: Total                                              209 MB/s | 3.5 MB  00:00     
c910f03c09k11: Retrieving key from file:///root/xcat-core/repodata/repomd.xml.key
[c910f03c09k09]: c910f03c09k11: warning: /root/xcat-core/perl-xCAT-2.14.3-snap201808090615.noarch.rpm: Header V4 RSA/SHA1 Signature, key ID ca548a47: NOKEY
[c910f03c09k09]: c910f03c09k11: Importing GPG key 0xCA548A47:
c910f03c09k11:  Userid     : "xCAT Automatic Signing Key <xcat@cn.ibm.com>"
c910f03c09k11:  Fingerprint: f2a2 306f 7eb3 3463 487f 7b84 6858 3856 ca54 8a47
c910f03c09k11:  From       : /root/xcat-core/repodata/repomd.xml.key
c910f03c09k11: 
c910f03c09k11: 
c910f03c09k11: Public key for xCAT-genesis-scripts-x86_64-2.14.3-snap201808090615.noarch.rpm is not installed
c910f03c09k11: 
c910f03c09k11: 
c910f03c09k11:  Failing package is: 1:xCAT-genesis-scripts-x86_64-2.14.3-snap201808090615.noarch
c910f03c09k11:  GPG Keys are configured as: file:///root/xcat-core/repodata/repomd.xml.key
CHECK:rc == 0	[Failed]

```

**Comment by neo954** [comment]
The problem was fixed in pull request #5512.
